alica_engine/include/engine/BasicPlan.h

```diff
@@ -16,7 +16,7 @@ class Transition;
 
 struct PlanContext
 {
-    IAlicaWorldModel* worldModel;
+    Blackboard& worldModels;
```

Comment:
why is this variable named `worldModels`? Can't we just call it global blackboard?